README.md

```diff
@@ -248,11 +248,38 @@ A number of options are available to allow you to customise the SDK:
   </body>
   ```
 
-- **`isModalOpen {Boolean} optional`**
```

Comment:
We still want to have the `isModalOpen` documentation here

Reply:
whoops sorry

Reply:
fixed